### ISSUE TITLE
GraphQLError: Syntax Error: Expected Name, found "}"

### ISSUE THREAD

**Issue by denisoed** [VirtusLab-Open-Source/strapi-plugin-comments#79]
Hi.

Plugin version 1.0.2 gives an error.

![image](https://user-images.githubusercontent.com/17797700/137333233-00dff642-a476-413a-99b7-f100aa42a7eb.png)

OS: Linux Ubuntu 20.04
Node: v14.17.3
Yarn: 1.22.10

**Version 1.0.1 works fine**

**Comment by cyp3rius** [comment]
@denisoed provide please the plugin config you've got. Stack trace is too general to suggest something.

**Comment by diegoaguilar** [comment]
@cyp3rius this is also happening for me using the 3.6 Strapi version and everything yarn installed

**Comment by wdcarlos** [comment]
i think its related to graphql support. Downgrade to 1.0.1 makes server boot again but cant see comments collection, just the infinity loading screen.

**Comment by rerecampos** [comment]
I had same error, without using GQL provider option, just upgrading to 1.0.2 version.
 - **1.0.1** still works

Strapi: v3.6.8
Plugin: v1.0.2
OS: Windows 10 Pro
Node: v14.16.1
Strapi GQL plugin: v3.6.8

Current custom.js config: 
 - Folder path: ~ root ~/config/custom.js
```
module.exports = {
  plugins: {
    comments: {
      enableUsers: false,
      badWords: false,
    },
  },
};
```

I've already tried cleaning node_modules, installed everything again, ran the build on the strapi, ran in develop mode and still not working

![image](https://user-images.githubusercontent.com/82174766/140217758-59e57daa-76d1-46b3-b65f-9a38ace530ba.png)

**Comment by cyp3rius** [comment]
Have you tried to follow: https://github.com/VirtusLab-Open-Source/strapi-plugin-comments#additional-gql-configuration ?

**Comment by rerecampos** [comment]
> Have you tried to follow: https://github.com/VirtusLab-Open-Source/strapi-plugin-comments#additional-gql-configuration ?

Yes, I've tried it with and without this GQL configuration, both give the same error mentioned above

**Comment by ltsNotMike** [comment]
I think the problem might be connected to the docs. The configuration for GraphQL should be inside config/plugins.js instead of config/custom.js. We have a pull request ready #87 to update the docs and migrate all configurations into one file (plugins.js). 

Example of plugins.js
```js
module.exports = {
  comments: {
    relatedContentTypes: {
      movies: {
        uuid: 'application::movies.movies',
        contentManager: true,
        __contentType: '',
        key: 'title',
        value: 'id',
      }
    }
  },
}
```

**Comment by rerecampos** [comment]
> I think the problem might be connected to the docs. The configuration for GraphQL should be inside `config/plugins.js` instead of `config/custom.js`. We have a pull request ready #87 to update the docs and migrate all configurations into `plugins.js`.
> 
> #### Example of plugins.js
> ```js
> module.exports = {
>   comments: {
>     relatedContentTypes: {
>       movies: {
>         uuid: 'application::movies.movies',
>         contentManager: true,
>         __contentType: '',
>         key: 'title',
>         value: 'id',
>       }
>     }
>   },
> }
> ```

Thanks! With this configuration the strapi was built, ran without problems and appeared in GQL!

**Comment by mykoshka** [comment]
@ltsNotMike Yap that worked for me too! I did have some issues, when trying to install directly in my existing project. So I started a new project rebuild the apis and exported -> important the data. Now it works perfectly...

Thanks

**Comment by cyp3rius** [comment]
Change of configuration source has been released as part `1.0.4`.
